src/app/contextBroker/contextBroker.cpp

```diff
@@ -368,6 +368,8 @@ PaArgument paArgs[] =
 
   { "-logDeprecate",                &logDeprecate,          "LOG_DEPRECATE",            PaBool,   PaOpt, false,                           false, true,                  LOG_DEPRECATE_DESC           },
 
+  { "-maxthreshold",                &maxthreshold,          "MAX_THRESHOLD",            PaInt,    PaOpt, 0,                               PaNL,  PaNL,                  MAX_THRESHOLD_DESC           },
```

Comment:
`maxthreshold` is too ambiguous (threshold on what?). The name should tell it is a max threshold on notification queues.

Maybe something like this:

```suggestion
  { "-notifAlarmThreshold",                &maxthreshold,          "MAX_THRESHOLD",            PaInt,    PaOpt, 0,                               PaNL,  PaNL,                  MAX_THRESHOLD_DESC           },
```

Reply:
Updated in https://github.com/telefonicaid/fiware-orion/pull/4375/commits/1ebccb65531531a8dc0856d8de499b6a34d5a733